src/style.css

```diff
@@ -0,0 +1,66 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+.search-icon {
+  position: absolute;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+  z-index: 1;
+}
+
+.search-icon--small {
+  height: 11px;
+  left: 13px;
+  top: 15px;
+}
+
+.box {
+  display: block;
+  position: relative;
+  margin-top: 20px;
```

Comment:
Move margin-top to form (add class for the form), the whole form has margin-top not only the label